### ISSUE TITLE
KeyError: 'arguments' when working with streamlit

### ISSUE THREAD

**Issue by sergei-grechanik** [davidhalter/jedi#1574]
The following code results in `KeyError: 'arguments' `:
```
import jedi
jedi.Script("import streamlit; streamlit.altair_chart.").complete()
```

Python 3.6.9 
streamlit 0.58.0 (installed from pip)

Stack trace: https://gist.github.com/sergei-grechanik/9f27cb4e5efdd4f4ba3d9a16b1a77e4f

**Comment by davidhalter** [comment]
Thanks for the report. Definitely a bug.

**Comment by davidhalter** [comment]
Fixed in 6dbc5e783e9d61c825c059ff114f163d183df4ea.
